lib/yoti/http/payloads/aml_profile.rb

```diff
@@ -0,0 +1,41 @@
+module Yoti
+  # Manages the AML check Profile object
+  class AmlProfile
+    def initialize(given_names, family_name, aml_address, ssn = nil)
+      @given_names = given_names
+      @family_name = family_name
+      @ssn = ssn
+      @address = aml_address
+
+      raise AmlError, 'The AML request requires given names, family name and an ISO 3166 3-letter code.' unless profile_valid
+      raise AmlError, 'Request for USA require a valid SSN and post code.' unless usa_valid
```

Comment:
Isn't it called Zip code in the US? Or are we just going to say postcode everywhere for consistency? 

(If we do stick with postcode, it is one word I believe)

Reply:
We'll use postcode as it is the more generic.

Reply:
I was just thinking that the only time we'll be showing this error will be for users in the US - so zip code might be better. But I guess if we call it zip code here and postcode elsewhere it'll be inconsistent